modules/applications/applications-nav.adoc

```diff
@@ -8,6 +8,13 @@
    **** xref:ui-builder/builder-declare-interface-in-bonita.adoc[Declare the interface into Bonita]
    **** xref:ui-builder/package-and-deploy-your-application.adoc[Package and deploy your application]
    ***** xref:ui-builder/production-packaging.adoc[Production packaging]
+  *** xref:ui-builder/version-control-with-git/version-control-with-git.adoc[Version control with Git]
+   **** xref:ui-builder/version-control-with-git/git-settings.adoc[Git settings]
+   **** xref:ui-builder/version-control-with-git/connect-git.adoc[Connect applications to Git]
+   **** xref:ui-builder/version-control-with-git/import-from-git.adoc[Import applications from Git]
+   **** xref:ui-builder/version-control-with-git/revert-changes.adoc[Revert changes]
+   **** xref:ui-builder/version-control-with-git/resolve-merge-conflicts.adoc[Resolve merge conflicts in Git]
+   **** xref:ui-builder/version-control-with-git/update-local-file-path.adoc[Update the local file path]
   *** xref:ui-builder/how-tos-builder.adoc[Guides]
```

Comment:
I wonder if we shouldn't move the `ui-builder/version-control-with-git/version-control-with-git.adoc[Version control with Git]` bloc and its sub-pages under `ui-builder/how-tos-builder.adoc[Guides]` instead

Reply:
Alright, sounds good